build.gradle

```diff
@@ -56,10 +56,6 @@ task sourcesJar(type: Jar, dependsOn:classes) {
 
 artifacts { archives sourcesJar }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = "4.6"
```

Comment:
Why not also set the new version here?

Reply:
Because the `wrapper` task already exists and this leads to a conflict.

```
* Where:
Build file '/home/lins/workspaces/github/timols/java-gitlab-api/build.gradle' line: 59

* What went wrong:
A problem occurred evaluating root project 'java-gitlab-api'.
> Cannot add task 'wrapper' as a task with that name already exists.
```

Actually, there is no need for it, the version is already defined in the `gradle/wrapper/gradle-wrapper.properties`.
And with an update, you actually specify the new version by parameter:

```
$ ./gradlew wrapper --gradle-version=5.5.1
```

Reply:
For reference: https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
Sigh. And suddenly Gradle 5.6 is there 🤣